CMakeLists.txt

```diff
@@ -0,0 +1,88 @@
+hyro_init_package(signal_generator VERSION 0.0.1)
+
+
+hyro_find_package(hyro_utils REQUIRED)
+hyro_find_package(hyro_msgs REQUIRED)
+hyro_find_package(hyro_widgets REQUIRED)
+
+
+
+###########
+## Build ##
+###########
+
+## Specify additional locations of header files
+## Your package locations should be listed before other locations
+
+
+set(SIGNAL_GENERATOR_HEADERS
+  # add all *.h files here
+  include/hyro/SignalGeneratorComponent.hpp
+  include/hyro/DigitalConverterComponent.hpp
+
```

Comment:
Do not need empty lines here. If you want to separate the headers of the components from the headers of the messages, you can add a comment between them.